src/components/formWrapper/FormWrapper.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div>
+    <div v-if="false">
```

Comment:
i think we don't need the v-if it has a harcoded value

Reply:
removed